### PR TITLE
six, bump, add pyversions

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-python/six/six-1.12.0.recipe [hunk 1]
```diff
@@ -41,6 +41,10 @@ BUILD_PREREQUIRES="$BUILD_PREREQUIRES
 	cmd:python$pythonVersion"
 done
 
+REPLACES_python="
+	python_six
+	"
```
Comment (korli) [inline dev-python/six/six-1.12.0.recipe]: Any reason to add a REPLACES_python? If none, please remove as always.

Comment (Begasus) [reply dev-python/six/six-1.12.0.recipe]: Haven't been checking up on these, I used the same base as for the other recipes, as far as I can see they build ok for all python versions?

Comment (korli) [reply dev-python/six/six-1.12.0.recipe]: Did python_six ever exist?

**Comment by Begasus** [comment]
Was on my way but you beat me to it :) :+1:

**Pull request merged**